base/commands/serializer/compact_class.go

```diff
@@ -0,0 +1,49 @@
+package serializer
+
+import (
+	"fmt"
+	"io/fs"
+	"os"
+	"path"
+	"strings"
+)
+
+func generateCompactClasses(lang string, schema Schema) (map[ClassInfo]string, error) {
+	classes := make(map[ClassInfo]string)
```

Comment:
Could you move this after the switch?

Reply:
`classes` is used inside switch

Reply:
Yeah, right. `generateJavaClasses` is a strange function which updates a map instead of returning one. It's better to change `generateJavaClasses` so it returns a `map`.